airflow/task/task_runner/standard_task_runner.py

```diff
@@ -105,9 +105,16 @@ def return_code(self, timeout=0):
         return self._rc
 
     def terminate(self):
+        self.log.debug("self.process: %r", self.process)
         if self.process is None:
             return
 
+        # Reap the child process - it may already be finished
+        self.log.debug("process.is_running(): %r, status(): %r", self.process.is_running(), self.process.status())
+        self._rc = os.waitpid(self.process.pid)
+        #self._rc = self.process.wait(timeout=0)
```

Comment:
```suggestion
```

Reply:
oh whoops, don't think I meant to include these changes at all

Reply:
Yeah these were just debugging left over, not needed.

Reply:
In part.